subt/doc/virtual-howto.md

```diff
@@ -22,29 +22,42 @@ docker pull osrf/subt-virtual-testbed:cloudsim_sim_latest
 docker pull osrf/subt-virtual-testbed:cloudsim_bridge_latest
 ```
 
+## Precoditions
+
+3D X server needs to be running under the same user that is going to be running the simulation.
+If the X server runs display `:0` the variable `DISPLAY` needs to be set and exported
+```
+export DISPLAY=:0
+```
+Also `XAUTHORITY` variable needs to be set to point to the location of the XAuthority file. Example on
+ubuntu could be
+```
+$ echo $XAUTHORITY
+/run/user/1000/gdm/Xauthority
+```
+Another possiblity is to run `xhost +` to disable the security on the X server.
```

Comment:
actually should not this be original "xhost +local:root" ... but maybe + is ok

Reply:
as I am not using it, I don't know - so I'll keep the original variant

Reply:
According to the man page `xhost +` allows anyone to connect so it is not the wrong thing to do (it should be enough).

When I run plain `xhost` I get this output
```
$ xhost
access control enabled, only authorized clients can connect
SI:localuser:zbynek
```
After running `xhost +local:root` I get
```
$ xhost +local:root
non-network local connections being added to access control list
$ xhost
access control enabled, only authorized clients can connect
LOCAL:
SI:localuser:zbynek
```
so that works but it does not do what is suggested. You probably wanted
```
$ xhost si:localuser:root
localuser:root being added to access control list
```
but this whole thing should be obsolete once we start using #747 

What we really want is most likely this:
```
$ xhost +local:
non-network local connections being added to access control list
```
as that allows our solution and also the simulation access to the xserver (they run as different users).

Reply:
done in  dfd6fd8